src/node/rpc/memberfrontend.h

```diff
@@ -66,6 +66,17 @@ namespace ccf
            nodes->put(id, *info);
            return true;
          }},
+        // accept a node
```

Comment:
comment should say "accept new code" maybe?

Reply:
Thanks :)